fusor/fusor.py

```diff
@@ -281,12 +290,18 @@ def critical_domain(
         if not gene_descr:
             return None, warning
 
+        loc_descr = self._location_descriptor(
+            start, end, sequence_id,
+            seq_id_target_namespace=seq_id_target_namespace
+        )
+
```

Comment:
If critical domain locations should only be on the protein sequence, we should do validation checks.

Reply:
That's a really good point -- I think technically it can be any of the above but our other tooling will definitely only support protein coordinates so I'll constrain it now and we can remove it later if necessary?